tensorhive/controllers/restriction/get_restriction_controller.py

```diff
@@ -0,0 +1,72 @@
+from flask_jwt_extended import jwt_required
+from tensorhive.models.Restriction import Restriction
+from tensorhive.models.User import User
+from tensorhive.models.Group import Group
+from tensorhive.models.Resource import Resource
+from tensorhive.models.RestrictionSchedule import RestrictionSchedule
+from sqlalchemy.orm.exc import NoResultFound
+from tensorhive.config import API
+import logging
+
+log = logging.getLogger(__name__)
+R = API.RESPONSES['restriction']
+G = API.RESPONSES['general']
+
+
+@jwt_required
+def get_all():
+    return [
+        restriction.as_dict(include_groups=True, include_users=True,
+                            include_resources=True) for restriction in Restriction.all()
+    ], 200
+
+
+@jwt_required
+def get_selected(user_id, include_user_groups, group_id, resource_id, schedule_id):
+    try:
+        include_groups = True
+        include_users = True
+        include_resources = True
```

Comment:
you can simplify it a bit:
```suggestion
        include_groups = group_id is None
        include_users = user_id is None
        include_resources = resource_id is None
```

Also it might be worth adding an explanatory comment here as to why we're doing this (excluding certain parts from the response...)